React/app/components/DonationList/index.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * HomePage
+ *
+ * This is the first thing users see of our App, at the '/' route
+ *
+ */
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
So if you don't have the useTheme imports you don't have to have the theme variable around below. (inside of your useStyles variable) I would try getting rid of theme in the useStyles (func won't need to be curried) and set bgColor manually.